ddtrace/appsec/_iast/_handlers.py

```diff
@@ -190,7 +190,8 @@ def _on_django_func_wrapped(fn_args, fn_kwargs, first_arg_expected_type, *_):
             return
 
         http_req = fn_args[0]
-        set_iast_request_endpoint(http_req.method, http_req.resolver_match.route)
+        if http_req.resolver_match is not None:
```

Comment:
necessary change?

Reply:
Well, it was needed on riot, yup[ [raw log without this]. ](https://gitlab.ddbuild.io/DataDog/apm-reliability/dd-trace-py/-/jobs/1012514685/raw).

What wasn't needed was (if current run finish correctly) the remote config enablement.